tests/e2escenarios/e2e_images_test.go

```diff
@@ -85,15 +86,11 @@ var _ = Describe("odo supported images e2e tests", func() {
 		Expect(cmpLst).To(ContainSubstring("Not Pushed"))
 	}
 
-	Context("odo supported images deployment", func() {
-		It("Should be able to verify the openjdk18-openshift image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "java:8", project)
-			verifySupportedImage(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "openjdk", "java:8", project, appName, context)
-		})
-
-		It("Should be able to verify the openjdk-11-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "java:8", project)
-			verifySupportedImage(filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "openjdk", "java:8", project, appName, context)
+	Context("odo supported images deployment on x86", func() {
+		JustBeforeEach(func() {
+			if runtime.GOARCH != "amd64" {
```

Comment:
@zhengxiaomei123 As per the context it says supported image deployment on x86 architecture but in the condition it check for the opposite (runtime.GOARCH != "amd64" -> x86), i mean its skipping the test for same x86 architecture.

May be you need to update the context description like ```odo supported images deployment on amd64```

Reply:
> @zhengxiaomei123 As per the context it says supported image deployment on x86 architecture but in the condition it check for the opposite (runtime.GOARCH != "amd64" -> x86), i mean its skipping the test for same x86 architecture.
> 
> May be you need to update the context description like `odo supported images deployment on amd64`

Sure. I will update it. I confused those two arch.